plugin/pkg/scheduler/factory/factory.go

```diff
@@ -335,14 +336,18 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 				return nil, err
 			} else {
 				extenders = append(extenders, extender)
+				// Check if bind is offloaded to the extender.
+				if policy.ExtenderConfigs[ii].BindVerb != "" {
+					binder = extender
+				}
 			}
 		}
 	}
-	return f.CreateFromKeys(predicateKeys, priorityKeys, extenders)
+	return f.CreateFromKeys(predicateKeys, priorityKeys, extenders, binder)
 }
 
 // Creates a scheduler from a set of registered fit predicate keys and priority keys.
-func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String, extenders []algorithm.SchedulerExtender) (*scheduler.Config, error) {
+func (f *ConfigFactory) CreateFromKeys(predicateKeys, priorityKeys sets.String, extenders []algorithm.SchedulerExtender, extBinder scheduler.Binder) (*scheduler.Config, error) {
```

Comment:
Sure, but you could still do what I said. In this function, you loop over the array of `extenders` to find if any of them is a binder. When `CreateFromKeys` is called from `CreateFromProvider`, the array will be empty and your code will not find any binder.